zombies.js

```diff
@@ -224,7 +297,13 @@
  * @property {number} speed
  * @property {boolean} isAlive      Default value should be `true`.
  */
-
+function  Zombie (health, strength, speed) {
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  var maxHealth = health;
```

Comment:
Private variables have a specific naming convention, it is not being used here. Also, the way it is currently setup, the private variable is not accessable outside of the constructor.